Framework/API/src/IFunction.cpp

```diff
@@ -1584,6 +1584,12 @@ void IFunction::sortTies() {
     if (!tie) {
       continue;
     }
+    // Ignore height ties for Gaussian peaks as component of a CrystalFieldFunction
+    if (this->name() == "CrystalFieldFunction" && tie->ownerFunction()->name() == "Gaussian" &&
```

Comment:
Could I ask if there would be a different way to solve this that doesn't required quite detailed knowledge of the function/tie specifics in the base class?

Maybe an `bool IFunction::ignoreTie(const Tie&)` that a function could override that by default returned `false`?